common.php

```diff
@@ -88,7 +88,7 @@ public function loadConfig() {
         $sql = "SELECT l.sql_log, l.ts
 			FROM redcap_log_event l WHERE
 		 		l.project_id = " . intval($this->project_id) . "
-			AND l.page = 'PLUGIN'
+			AND (l.page = 'PLUGIN' OR l.page LIKE '%autonotify2%')
```

Comment:
Allows config to be reloaded from the log. Based on code from 123andy/redcap-plugin-autonotify#4